.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!--
+    If you are changing an existing module or resource, describe why that configuration is no longer acceptable. If you are fixing a misconfiguration, make sure to describe both the intended result and the actual (faulty) result.
```

Comment:
Maybe suggest having a test for the original failure, that now passes with the change?

Reply:
This is IaC so there are typically no tests.